src/cryptonote_core/cryptonote_tx_utils.h

```diff
@@ -117,7 +117,7 @@ namespace cryptonote
   };
 
   //---------------------------------------------------------------
-  crypto::public_key get_destination_view_key_pub(const std::vector<tx_destination_entry> &destinations, const boost::optional<cryptonote::account_public_address>& change_addr);
+  crypto::public_key get_payment_id_encr_dest_view_key_pub(const std::vector<tx_destination_entry> &destinations, const boost::optional<cryptonote::account_public_address>& change_addr);
```

Comment:
Could we please get a comment here describing the overall purpose of the function since we're renaming?

Reply:
That's basically the point of proper function naming, where possible, i.e. in this case I think we can get away without commenting the function, but I would be open to adding one?

Reply:
The function is named much better, and Robert C. Martin promotes such kind of naming in his books (*), but a bit of more human readable text wouldn't hurt I think.

(*) A digression: he'd also definitely say, that such a function has to be broken into many smaller functions, that *do one thing* each

Reply:
Since we're talking about it, I'd be happy to take a crack at splitting this function into its two (IMO) factors. Actually should be fairly easy. But it will create a higher overhead in review on a highly critical codepath. But.. also, hopefully is fairly trivial and hopefully has enough qualified eyes on reviewing it.

Reply:
Exactly for that reason, I think a separate PR would increase the chance of success :)

Reply:
Ok, cool. I will leave it for later then.